lib/dotcom/trip_plan/related_link.ex

```diff
@@ -14,7 +14,10 @@ defmodule Dotcom.TripPlan.RelatedLink do
             icon_name: nil
 
   @stops_repo Application.compile_env!(:dotcom, :repo_modules)[:stops]
-  @default_opts [route_by_id: &Routes.Repo.get/1, stop_by_id: {@stops_repo, :get_parent, 1}]
+  @default_opts [
+    route_by_id: &Routes.Repo.get/1,
+    stop_by_id: Function.capture(@stops_repo, :get_parent, 1)
```

Comment:
All of these default ops can be removed

Reply:
I removed the stop one, but the routes repo isn't clean up yet, so I'd like to leave that in here and change the rest of this when we get to that point.